create-publication-from-scopus/src/main/java/no/sikt/nva/scopus/ScopusSourceType.java

```
@@ -0,0 +1,22 @@
+package no.sikt.nva.scopus;
+
+import java.util.Arrays;
+
+public enum ScopusSourceType {
+
+    JOURNAL("j");
+
+    public final String code;
+
+    ScopusSourceType(String code) {
+        this.code = code;
+    }
+
+    public static ScopusSourceType valueOfCode(String code) {
+        return Arrays.stream(values())
+                .filter(value -> value.code.equals(code))
+                .findAny()
+                .orElse(null);
```

better to throw an exception here instead of returning null?